src/lib/core/overlay/overlay.ts

```diff
@@ -52,10 +52,21 @@ export class Overlay {
    * Creates the DOM element for an overlay and appends it to the overlay container.
    * @returns Promise resolving to the created element.
    */
-  private _createPaneElement(): HTMLElement {
+  private _createPaneElement(state: OverlayState): HTMLElement {
     var pane = document.createElement('div');
-    pane.id = `md-overlay-${nextUniqueId++}`;
+    const id = nextUniqueId++;
+    pane.id = `md-overlay-${id}`;
     pane.classList.add('md-overlay-pane');
+    pane.classList.add(`md-has-index-${id}`);
```

Comment:
So you can more granularly control stacking order of multiple panes inside a single overlay.. I guess you could do the same with `:nth-child`.. but you know, with BEM and everything having a class could be handy.

I won't mind removing it though, you tell me.